codalab/model/tables.py

```diff
@@ -242,7 +242,7 @@
     Column('affiliation', String(255, convert_unicode=True), nullable=True),
     Column('url', String(255, convert_unicode=True), nullable=True),
     # Quotas
-    Column('time_quota', Float, nullable=False),  # Number of seconds allowed
+    Column('parallel_job_quota', Integer, nullable=False),  # Number of parallel jobs allowed
```

Comment:
Wondering if we should name this `parallel_run_quota` to avoid introducing a new term `job`